aiogram/types/message.py

```diff
@@ -957,7 +957,10 @@ async def answer_location(
         self,
         latitude: base.Float,
         longitude: base.Float,
+        horizontal_accuracy: typing.Optional[base.Float] = None,
         live_period: typing.Optional[base.Integer] = None,
+        heading: typing.Optional[base.Integer] = None,
+        proximity_alert_radius: typing.Optional[base.Integer] = None,
```

Comment:
This change will break a code like this:
```python3
await message.answer_location(42.0, 42.0, 60, True)
```
While originally it was assumed to be
```python3
await message.answer_location(latitude=42.0, longitude=42.0, live_period=60, disable_notification=True)
```
it will be (after applying this patch)
```python3
await message.answer_location(latitude=42.0, longitude=42.0, horizontal_accuracy=60, live_period=True)
```

@JrooTJunior what do you think about this?